tests/openQA_containers/multiple_container_webui.pm

```diff
@@ -0,0 +1,74 @@
+ use strict;
+ use base "openQAcoretest";
+ use testapi;
+ 
+ sub wait_for {
+     my ($container) = @_;
+     my $count = 30;
+     while (system("docker logs $container 2>&1 | grep Listening >/dev/null") != 0) {
+         sleep 1;
+         $count = $count - 1;
+         last if ($count == 0);
+     }
+ 
+     system("docker logs $container 2>&1 | grep Listening >/dev/null");
+ }
+ 
+ sub run {
+     my ($self) = @_;
+     my $volumes = "-v \"/root/data/factory:/data/factory\" -v \"/root/data/tests:/data/tests\" -v \"/root/openQA/container/webui/conf:/data/conf:ro\"";
+ 
+     assert_script_run("mkdir -p /root/data/factory/{iso,hdd,other} /root/data/tests");
+     assert_script_run("docker network create testing");
+     assert_script_run("docker run -d --network testing -e POSTGRES_PASSWORD=openqa -e POSTGRES_USER=openqa -e POSTGRES_DB=openqa --net-alias=db --name db postgres", timeout => 600);
```

Comment:
do you think that `--rm` or `--restart` could be used here? mainly restart because you want the db to restart on exit??

Reply:
Done